src/tools/wasm-ctor-eval.cpp

```diff
@@ -369,7 +369,10 @@ int main(int argc, const char* argv[]) {
       .add("--ctors", "-c", "Comma-separated list of global constructor functions to evaluate",
            Options::Arguments::One,
            [&](Options* o, const std::string& argument) {
-             ctorsString = argument;
+               if (!argument.empty() && argument[0] == '@')
```

Comment:
thanks for the comment. the indentation is caused by IDE settings required by our company and I have changed to fit the code standards on your side.  
The response file is the default option for our team here as we have thousands of global ctors in the system...  and we prefer to have another PR for the option work.